pkg/scheduler/framework/plugins/nodeports/node_ports.go

```diff
@@ -97,6 +98,14 @@ func getPreFilterState(cycleState *framework.CycleState) (preFilterState, error)
 	return s, nil
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *NodePorts) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Node, ActionType: framework.Add | framework.UpdateNodeCondition},
```

Comment:
Why UpdateNodeCondition?

What about Pod deletions?

Reply:
@alculquicondor Thank you for your review.
1. I refer to [this document](https://kubernetes.io/zh/docs/concepts/architecture/nodes/#condition) . It mentioned that changes in node network configuration will lead to changes in node status (`NetworkUnavailable`), so I added framework.UpdateNodeCondition, is my understanding correct?😊 I am considering whether it should be modified to `framework.UpdateNodeAllocatable`.
2. The event of pod being deleted is missed by me, I shall add it.
Thanks again for your suggestions and help.

Reply:
The node ports plugin doesn't consider network availability, only which ports are occupied.